Makefile

```diff
@@ -14,6 +14,7 @@ VERSION_PKG ?= "github.com/jaegertracing/jaeger-operator/pkg/version"
 JAEGER_VERSION ?= "$(shell grep -v '\#' jaeger.version)"
 OPERATOR_VERSION ?= "$(shell git describe --tags)"
 STORAGE_NAMESPACE ?= "${shell kubectl get sa default -o jsonpath='{.metadata.namespace}' || oc project -q}"
+ES_OPERATOR_NAMESPACE = openshift-logging
```

Comment:
I'd change this so that you don't collide with logging components

Reply:
The question is if `oc create -f es-operator.yml` would work as it has hardcoded namespace. We don't deploy any logging in e2e tests so it should be fine.

Reply:
I think this shold not matter. There should be only one es-operator anyways. We are not deploying ES instance to this namespace.

Reply:
AFAIK You'll need your own manifest for deploying the operator to onboard it to operatorhub.

Reply:
What manifests do you mean? Our manifests are in `deploy` directory. There is also yamls for `olm-catalog` btw. https://operatorhub.io/operator/jaeger-operator.v1.8.2

Reply:
es-operator with your desired namespace, envvars, etc...